docs/intro.rst

```diff
@@ -16,6 +16,11 @@ them to *.mpy* files. These source files can also be
 `frozen <https://docs.micropython.org/en/latest/develop/optimizations.html?highlight=frozen#frozen-bytecode>`_
 and incorporated into a custom MicroPython firmware.
 
+Alternatively you can install using `mpremote` and `mip`
+```
+mpremote a0 mip install https://github.com/miguelgrinberg/microdot/releases/download/v1.3.3/microdot.mpy
```

Comment:
this method of installation is flawed, because mpy files cannot be used across MicroPython versions. When new MicroPython releases are out the mpy files that were archived in the release will not work anymore. Basically you need a set of mpy files for each MicroPython release.

Reply:
Updated to use include the micropython version